### PR TITLE
make str log fields consistent

### PR DESCRIPTION
**Pull request by juliaElastic**
## What is the problem this PR solves?

// Please do not just reference an issue. Explain WHAT the problem this PR solves here.

Making log fields consistent to make it easier to check the logs.

## How does this PR solve the problem?

// Explain HOW you solved the problem in your code. It is possible that during PR reviews this changes and then this section should be updated.

## Design Checklist

- [ ] I have ensured my design is stateless and will work when multiple fleet-server instances are behind a load balancer.
- [ ] I have or intend to scale test my changes, ensuring it will work reliably with 100K+ agents connected.
- [ ] I have included fail safe mechanisms to limit the load on fleet-server: rate limiting, circuit breakers, caching, load shedding, etc.

## Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `./changelog/fragments` using the [changelog tool](https://github.com/elastic/fleet-server#changelog)

## Related issues

Closes https://github.com/elastic/fleet-server/issues/3239

### CODE REVIEW

**Comment by michel-laterman** [comment]
thanks for getting a start on this!

File: internal/pkg/coordinator/monitor.go [hunk 1]
```diff
@@ -431,7 +431,7 @@ func runCoordinatorOutput(ctx context.Context, cord Coordinator, bulker bulk.Bul
 			if err != nil {
 				s.Err(err).Msg("Policy coordinator failed to add a new policy revision")
 			} else {
-				s.Info().Int64("revision_id", p.RevisionIdx).Msg("Policy coordinator added a new policy revision")
```
Comment (juliaElastic) [inline internal/pkg/coordinator/monitor.go:434]: removed redundant log field (already there in line 429)

File: internal/pkg/api/handleCheckin.go [hunk 2]
```diff
@@ -765,8 +766,8 @@ func processPolicy(ctx context.Context, zlog zerolog.Logger, bulker bulk.Bulk, a
 	defer span.End()
 	zlog = zlog.With().
 		Str("fleet.ctx", "processPolicy").
-		Int64("fleet.policyRevision", pp.Policy.RevisionIdx).
-		Int64("fleet.policyCoordinator", pp.Policy.CoordinatorIdx).
+		Int64(dl.FieldRevisionIdx, pp.Policy.RevisionIdx).
```
Comment (michel-laterman) [inline internal/pkg/api/handleCheckin.go:769]: Should we define these as constants in `logger/ecs.go`?

Comment (juliaElastic) [reply internal/pkg/api/handleCheckin.go]: done

**Review by michel-laterman — APPROVED** [review]

**Pull request merged**